README.md

```diff
@@ -65,9 +65,9 @@ pip install -e ".[test,doc]"
 ### Copyright
 
 The mdaencore source code is hosted at https://github.com/MDAnalysis/mdaencore
-and is available under the GNU General Public License, version 2 (see the file [LICENSE](https://github.com/MDAnalysis/mdaencore/blob/main/LICENSE)).
+and is available under the GNU General Public License, version 2 or any later version (see the file [LICENSE](https://github.com/MDAnalysis/mdaencore/blob/main/LICENSE)).
```

Comment:
Need to make clear that we allow "or any later version".

Reply:
Yes that is correct.

We should make it clear that the package is also released under GPLv3-or-later, since we are relying on MDAnalysis which itself is packaged under GPLv3-or-later.